app/views/components/_published-dates.html.erb

```diff
@@ -6,7 +6,7 @@
   link_to_history ||= false
 %>
 <% if published || last_updated %>
-<div class="app-c-published-dates" <% if history.any? %>id="history" data-module="toggle"<% end %>>
+<div <% if history.any? %>id="history" data-module="toggle" class="app-c-published-dates app-c-published-dates--history" <% else %> class="app-c-published-dates" <% end %>>
```

Comment:
I feel like there's a bit of repetition here with the classes, it could perhaps be simplified something like this:

`history_class = "app-c-published-dates--history" if history.any?`
`<div <% if history.any? %>id="history" data-module="toggle"<% end %> class="app-c-published-dates <%= history_class %>">`